robots/pkg/flakefinder/downloader_test.go

```diff
@@ -3,9 +3,8 @@ package flakefinder_test
 import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
-	"github.com/google/go-github/v28/github"
 	"kubevirt.io/project-infra/robots/pkg/flakefinder"
+	github2 "kubevirt.io/project-infra/robots/pkg/flakefinder/github"
```

Comment:
nit: I find the `github2` ugly and a bit confusing, since the reader has to keep in mind that they are looking at usage of the GitHub implementation for the "repo query api" so to say. I understand that you want to keep it consistent though.

So, since I can't come up with a better solution for this right now myself I guess this improvement is for a later PR.

Reply:
Done. Renamed to `ghapi`.